x/stake/keeper.go

```diff
@@ -343,8 +356,7 @@ func (k Keeper) GetParams(ctx sdk.Context) (params Params) {
 	store := ctx.KVStore(k.storeKey)
 	b := store.Get(ParamKey)
 	if b == nil {
-		k.params = defaultParams()
-		return k.params
+		panic(errors.New("Stored params should not have been nil"))
```

Comment:
do not need to use `errors.New` here (https://play.golang.org/p/skq8dkMt8fd)

Reply:
Thanks, fixed.